map/map_tests/bookmarks_test.cpp

```diff
@@ -1499,7 +1499,25 @@ UNIT_CLASS_TEST(Runner, ExportSingleUnicode)
     TEST(base::DeleteFileX(kmz), ());
     TEST(base::DeleteFileX(tmpPath), ());
   };
-  bmManager.PrepareFileForSharing(std::move(categories), checker);
+  bmManager.PrepareFileForSharing(std::move(categories), checker, KmlFileType::Text);
+}
+
+UNIT_CLASS_TEST(Runner, ExportSingleGpx)
+{
+  string file = GetPlatform().TestsDataPathForFile("gpx_test_data/route.gpx");
+  BookmarkManager bmManager(BM_CALLBACKS);
+  bmManager.EnableTestMode(true);
+  BookmarkManager::KMLDataCollection kmlDataCollection;
+  kmlDataCollection.emplace_back(file, LoadKmlFile(file, KmlFileType::Gpx));
+  bmManager.CreateCategories(std::move(kmlDataCollection));
+  auto categories = bmManager.GetUnsortedBmGroupsIdList();
+  auto checker = [](BookmarkManager::SharingResult const & result)
+  {
+    auto fileName = result.m_sharingPath;
+    TEST(fileName.find("Some random route.gpx") != std::string::npos, ());
```

Comment:
TEST_EQUAL 0?

Reply:
I didn't get what should be compared with 0? fileName is "/tmp/Some random route.gpx" in this test

Reply:
I see, that's confusing, then it should be renamed to filePath.